etl/steps/data/garden/smoking/2024-05-30/cigarette_sales.meta.yml

```diff
@@ -0,0 +1,115 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
```

Comment:
That's an open discussion. I had a strict criterion on this, but we couldn't reach an agreement. In many cases, the data is identical to the original, so that should be mino processing. In this case, the data is not identical, for example due to the processing of historical regions like Germany. So that to me is enough to call it major. But again, there are no clear rules for now on this choice.

Reply:
okay, that makes sense to me! Will follow that logic in the future!